appeals/web/src/server/appeals/appeal-details/lpa-questionnaire/lpa-questionnaire.mapper.js

```diff
@@ -81,74 +95,12 @@ export async function lpaQuestionnairePage(lpaqDetails, appealDetails, currentRo
 		(/** @type {SummaryListRowProperties} */ row) => removeSummaryListActions(row)
 	);
 
-	/** @type {PageComponent[]} */
-	let appealTypeSpecificPageComponents = [];
-
-	switch (appealType) {
-		case 'Householder':
-			appealTypeSpecificPageComponents = householderLpaQuestionnairePage(
-				mappedLpaqDetails,
-				mappedAppealDetails
-			);
-			break;
-		default:
-			break;
-	}
-
 	/**
 	 * @type {PageComponent}
 	 */
 	const additionalDocumentsSummary = {
 		type: 'summary-list',
-		parameters: {
-			classes: 'pins-summary-list--fullwidth-value',
-			card: {
-				title: {
-					text: 'Additional documents'
-				},
-				actions: {
-					items:
-						(isFolderInfo(lpaqDetails.documents.lpaCaseCorrespondence)
-							? lpaqDetails.documents.lpaCaseCorrespondence.documents
-							: []
-						).length > 0
-							? [
-									{
-										text: 'Manage',
-										visuallyHiddenText: 'additional documents',
-										href: mapDocumentManageUrl(
-											lpaqDetails.appealId,
-											lpaqDetails.lpaQuestionnaireId,
-											(isFolderInfo(lpaqDetails.documents.lpaCaseCorrespondence) &&
-												lpaqDetails.documents.lpaCaseCorrespondence.folderId) ||
-												undefined
-										)
-									},
-									{
-										text: 'Add',
-										visuallyHiddenText: 'additional documents',
-										href: displayPageFormatter.formatDocumentActionLink(
-											lpaqDetails.appealId,
-											lpaqDetails.documents.lpaCaseCorrespondence,
-											buildDocumentUploadUrlTemplate(lpaqDetails.lpaQuestionnaireId)
-										)
-									}
-							  ]
-							: [
-									{
-										text: 'Add',
-										visuallyHiddenText: 'additional documents',
-										href: displayPageFormatter.formatDocumentActionLink(
-											lpaqDetails.appealId,
-											lpaqDetails.documents.lpaCaseCorrespondence,
-											buildDocumentUploadUrlTemplate(lpaqDetails.lpaQuestionnaireId)
-										)
-									}
-							  ]
-				}
-			},
-			rows: mappedLpaqDetails.lpaq.additionalDocuments.display.summaryListItems
-		}
+		parameters: { ...mappedLpaqDetails.lpaq.additionalDocuments.display.cardItem }
```

Comment:
would it be worth doing the null check here? some props might potentially be undefined

Reply:
It comes from a constant in the mapper file so wouldn't be null or undefined